internal/common/splunk/common.go

```diff
@@ -36,7 +36,7 @@ type AccessTokenPassthroughConfig struct {
 
 // Event represents a metric in Splunk HEC format
 type Event struct {
-	Time       float64                `json:"time"`                 // epoch time
+	Time       *float64               `json:"time,omitempty"`       // optional epoch time - set to nil if the event has a timestamp set to the start of epoch time
```

Comment:
Is it not more correct to say this?
```suggestion
	Time       *float64               `json:"time,omitempty"`       // optional epoch time - set to nil if the event timestamp is missing or unknown
```